### PR TITLE
accept all image files & reject images greater than 5Mb

### LINKED ISSUE #155: Support non-png image files

### PR DESCRIPTION
**Pull request by will-weiss**
Closes #155 and #154 

image

### CODE REVIEW

File: src/background/images.ts [hunk 1]
```diff
@@ -55,7 +55,14 @@ export async function takeScreenshot(
       targetId: tab.id,
     })
   } catch (error) {
-    dispatchBackgroundActions.imageCaptureFailure({ targetId: target.id, error })
+    console.error(error)
```
Comment (shachar-langer) [inline src/background/images.ts:58]: I think the `console.error` is redundant here

File: src/background/images.ts [hunk 2]
```diff
@@ -79,6 +96,13 @@ export async function imageUpload(targetId: FeedbackTargetId, file: File, dispat
       image: { type: 'imageupload', name: file.name, uri, blob },
     })
   } catch (error) {
-    dispatchBackgroundActions.imageCaptureFailure({ targetId, error })
+    console.error(error)
```
Comment (shachar-langer) [inline src/background/images.ts:99]: Same comment

File: src/background/responders.ts [hunk 3]
```diff
@@ -231,7 +233,7 @@ export const responders: Responders<Action> = {
       },
     }))
   },
-  imageUpload(state): Partial<StoreState> {
+  imageUpload(state, { file }): Partial<StoreState> {
```
Comment (shachar-langer) [inline src/background/responders.ts:236]: `file` is redundant.

**Comment by shachar-langer** [comment]
There's a weird empty gap between the status and the image when the 5MB alert pops up

**Comment by shachar-langer** [comment]
I was able to post a JPG image to my local server but not to the production server. When I try to post from the production, the spinner never stops.

File: src/background/images.ts (hunk 1 above)
Comment (will-weiss) [reply src/background/images.ts:54]: The failure doesn't include the stack trace, so useful to see that here for an error we really don't expect

**Review by shachar-langer — APPROVED** [review]

**Pull request merged**